source/2020-01-17-the-ember-times-issue-131.md

```diff
@@ -19,6 +21,33 @@ READMORE
 
 ---
 
+## [4 RFCs on improving Ember's autotracking and reactivity system 💬](https://github.com/emberjs/rfcs/blob/use-and-resources/text/0567-use-and-resources.md#introducing-use-and-resources)
+
+Ember Octane features a **new reactivity system**. In particular, **tracked properties** simplify syncing the DOM with JavaScript changes. (Find out how autotracking works from the [Ember Guides](https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/).)
+
+To help address some shortcomings, [Chris Garrett (@pzuraq)](https://github.com/pzuraq) proposed introducing the decorators and classes listed below. We encourage you to participate in RFCs and provide feedback today!
```

Comment:
Is `today` a hard deadline? Because it will also be on the website and then this might feel a bit weird. Maybe even discourage people if they feel that `today` has already passed.

Reply:
Heh, no. I guess I like to use the word today to suggest urgency, e.g. buy this car *today*! But yeah, the word's not always needed.

```suggestion
To help address some shortcomings, [Chris Garrett (@pzuraq)](https://github.com/pzuraq) proposed introducing the decorators and classes listed below. We encourage you to participate in RFCs and provide feedback!
```